python/cugraph/centrality/betweenness_centrality_wrapper.pyx

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cugraph.centrality.betweenness_centrality cimport betweenness_centrality as c_betweenness_centrality
+from cugraph.structure.graph_new cimport *
+from cugraph.utilities.column_utils cimport *
+from cugraph.utilities.unrenumber import unrenumber
+from libcpp cimport bool
+from libc.stdint cimport uintptr_t
+from libc.stdlib cimport calloc, malloc, free
+from cugraph.structure import graph_wrapper
+import cudf
+import cudf._lib as libcudf
+import rmm
+import numpy as np
+import numpy.ctypeslib as ctypeslib
+
+
+def betweenness_centrality(input_graph, normalized):
+    """
+    Call betweenness centrality
+    """
+
+    if not input_graph.adjlist:
+        input_graph.view_adj_list()
```

Comment:
Are datatype conversions taking place? Cugraph allows users to be able to enter datatypes incompatible with algorithms, like int64 vertices and int weights without worrying what the algorithm demands internally. Would that input still work as expected?

Reply:
We don't do anything with weights yet (although I will probably add that based on Alex' comments.

Do we create graphs with int64 vertices?  I thought we used renumber (which generates int32 vertices) for the vertex ids that were stored in the python graph object.

Reply:
OK, I have added datatype conversions.  I missed the significance of those when I was looking at the other code, I thought it was just for making CSR out of COO.